examples/sha256-hashes/sha256-hashes.sh

```diff
@@ -0,0 +1,15 @@
+# Running the program computes the hash and prints it in
+# a human-readable hex format.
+$ go run sha256-hashes.go
+sha256 this string
+1af1dfa857bf1d8814fe1af8983c18080019922e557f15a8a...
```

Comment:
`./tools/measure` won't let this line be longer than 54 characters, so I truncated the hash. Would you prefer I update `measure` to allow the full hash?

Reply:
Hmm, yes, it needs the size restriction to display it properly. Your workaround LGTM

Reply:
Sounds good, thanks!